android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -19,14 +22,33 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
     /**
      * Exported module name to JS.
      */
-    override fun getName() = "NativePlayerView"
+    companion object {
+        const val name = "NativePlayerView"
+    }
```

Comment:
Does some component need to access this constant, besides the `getName` function directly below?

If not, why the change? Moving it to a (private) top level property might be a better option (?)

Reply:
I created this constant so we could always reference the same name on both `getName` and `@ReactModule(name = ...)`, that's present at the top of the class. This last annotation is needed in order to make this module accessible from other native modules, and not necessarily by name, even though the annotation requires it.

Reply:
That's the same approach we used with the `PlayerModule` for example.

https://github.com/bitmovin/bitmovin-player-react-native/blob/development/android/src/main/java/com/bitmovin/player/reactnative/PlayerModule.kt#L9

Reply:
That's why we can't make it `private` (it will cause a compilation error).

Reply:
```
private const val MODULE_NAME = "PlayerModule"

@ReactModule(name = MODULE_NAME)
class PlayerModule(private val context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
    /**
     * In-memory mapping from `nativeId`s to `Player` instances.
     */
    private val players: Registry<Player> = mutableMapOf()

    /**
     * JS exported module name.
     */
    override fun getName() = MODULE_NAME
```

For the `PlayerModule` it could look like this. I think this would be a cleaner approach than the companion object :)

Reply:
Very similar for the `NativePlayerView` :)

That's what I meant with private top level property.

Reply:
Oh I see, that makes sense. I can update that, sure.

Reply:
Done cf4a0e628007495bb8d91fd28d0962067b11d5bd